core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -448,8 +453,9 @@ public void testListNamespaces() {
 
   @Test
   public void testListNestedNamespaces() {
-    Assumptions.assumeTrue(
-        supportsNestedNamespaces(), "Only valid when the catalog supports nested namespaces");
+    assumeThat(supportsNestedNamespaces())
+        .isEqualTo(true)
+        .as("Only valid when the catalog supports nested namespaces");
```

Comment:
`as` should be set before calling the assertion such as `isEqualTo`.

Reply:
Thank you for your suggestion! I will continue to improve this part of the code.